### ISSUE TITLE
Make the app responsive

### ISSUE THREAD

**Issue by Tynael** [rpuspana/myExpenses#1]
First of all, congratulations on creating this awesome app. I know it took hard work and perseverance to build it.

A great welcoming feature would be to make the app responsive for different desktop screen sizes and also mobile devices.

Best of luck!

**Comment by rpuspana** [comment]
Yeah it took a lot of work especially since I am just learning JS. Thank you, it means a lot coming from you... I know what you're saying and I agree. It shall be done my lord.
